### ISSUE TITLE
Test 4, Search button cannot be reached by keyboard

### ISSUE THREAD

**Issue by david-iterators** [CityOfBoston/Iterators#105]
Screengrab/videos (insert file names here)
2020 03 18, Test 4, Search button cannot be reached by keyboard.mp4

Link and Node #
https://d8-stg.boston.gov/

Type of User:  Anonymous

Environment
(Device > OS > Browser)
Windows Laptop > Windows 10 > Chrome

Trusted Tester Test
4. Keyboard Access and Focus

Brief Description
The user can access search functionality by clicking on the magnifying glass icon on the top right of the page. However, if the user attempts to focus the magnifying glass via Tab or Shift + Tab, focus skips it; users limited to keyboard access cannot close the menu.

Steps to reproduce
1.	00:00, Go to the home page and set focus to a header item before the magnifying glass
2.	00:03, Begin hitting the Tab key until the magnifying glass icon should be in focus
3.	00:07, Set focus to a body element, then use Shift + Tab to back up until the magnifying glass should be in focus

Desired Result
At least one of Steps 2, 3 brings focus to the search icon.

Actual Result
User unable to focus the search element using only the keyboard.
[2020 03 18, Test 4, Search button cannot be reached by keyboard.mp4.zip](https://github.com/CityOfBoston/Iterators/files/4370742/2020.03.18.Test.4.Search.button.cannot.be.reached.by.keyboard.mp4.zip)

**Comment by david-iterators** [comment]
Verified on https://d8-uat.boston.gov
